tests/queries/0_stateless/02115_rewrite_local_join_right_distribute_table.sql

```diff
@@ -0,0 +1,33 @@
+-- Tags: global, no-parallel
+CREATE DATABASE IF NOT EXISTS test_02115;
```

Comment:
External database is not required for this tests, you can use `currentDatabase()` for `Distributed` instead. And then you will not need any tags for this tests.